src/installer/pkg/projects/netcoreapp/sfx/Microsoft.NETCore.App.SharedFx.sfxproj

```diff
@@ -2,8 +2,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <SfxIdentity>Microsoft.NETCore.App</SfxIdentity>
```

Comment:
Does this break usage (https://github.com/dotnet/arcade/blob/866b2acd5ffc3c2031c102f6415fd7c6a1a370d5/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.sharedfx.targets#L97) or is this handled somewhere I'm not spotting?

Reply:
Ah I didn't realize it was being used in arcade as well. Reverting this change.